### PR TITLE
[12.0][MIG] purchase_reception_notify

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Standard migration

cc ~ @Eficent

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
Functional tests :+1:

File: purchase_reception_notify/data/mail.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```
Comment (nikul-serpentcs) [inline purchase_reception_notify/data/mail.xml]: @hveficent IMO `<odoo noupdate="1">`

File: purchase_reception_notify/models/stock_picking.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# Copyright 2018 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0).
```
Comment (nikul-serpentcs) [inline purchase_reception_notify/models/stock_picking.py]: `LGPL -> AGPL`

**Review by nikul-serpentcs — COMMENTED** [review]
Nitpicking

**Review by AdriaGForgeFlow — APPROVED** [review]
Code review :+1:

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM :+1:

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**